scripts/e2e-ios-build.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-xcodebuild -workspace ios/Rainbow.xcworkspace -scheme Rainbow -configuration Release -sdk iphonesimulator -derivedDataPath ~/Library/Developer/Xcode/DerivedData
+xcodebuild -workspace ios/Rainbow.xcworkspace -scheme Rainbow -configuration Release -sdk iphonesimulator -derivedDataPath ~/Library/Developer/Xcode/DerivedData/Build/Products/Release-iphoneos
```

Comment:
I think the path should be ~/Library/Developer/Xcode/DerivedData, since it didn't seem to have been able to use the cache (build is still 20+ min). But I think even better we can just remove this derivedDataPath flag since we're just gonna use the default location now.